package.json

```diff
@@ -5,36 +5,67 @@
   "scripts": {
     "build": "lerna run build --stream",
     "build:analyze": "ANALYZE_BUNDLE=true yarn build:single",
-    "build:demo": "yarn build && lerna run build:demo --stream",
+    "build:demo": "yarn build && yarn build:storybook && lerna run build:demo --stream",
     "build:single": "utils/scripts/scoped-npm-command.js --script build",
-    "format": "yarn format:package_json && yarn format:js && yarn format:markdown",
+    "build:storybook": "build-storybook -s ./utils/styleguide/public -o ./demo/storybook",
+    "format": "yarn format:package_json && yarn format:js && yarn format:ts && yarn format:markdown",
     "format:js": "prettier --loglevel warn --write 'packages/**/!(dist|umd)/*.js' && prettier --loglevel warn --write utils/**/*.js",
     "format:markdown": "prettier --loglevel warn --write 'packages/**/!(dist)/!(CHANGELOG)*.md'",
     "format:package_json": "prettier-package-json --write && lerna exec -- prettier-package-json --write",
+    "format:ts": "prettier --loglevel warn --write 'packages/**/!(dist|umd)/*.{ts,tsx}' && prettier --loglevel warn --write utils/**/*.{ts,tsx}",
     "postinstall": "lerna bootstrap",
     "lint": "yarn lint:css && yarn lint:js && yarn lint:markdown",
     "lint:css": "stylelint \"packages/*/src/**/*.js\" && stylelint \"utils/**/*.js\"",
-    "lint:js": "eslint packages/*/src/ utils/ --max-warnings 0",
+    "lint:js": "eslint packages/*/src/ utils/ .storybook/ --ext js,ts,tsx --max-warnings 0",
     "lint:markdown": "markdownlint API.md DEVELOPMENT.md README.md VERSIONING.md packages/*/src/**/*.md packages/*/src/*.md packages/*/README.md",
     "new": "utils/scripts/create-package.js",
     "prepare": "yarn build",
     "start": "utils/scripts/scoped-npm-command.js --script start",
     "start:demo": "yarn build:demo && live-server demo",
+    "start:storybook": "start-storybook -s ./utils/styleguide/public -p 6006",
```

Comment:
Just to note, both `start:storybook` and `build:storybook` will eventually go away as they become the new `start:demo` and `build:demo`.